standalone/instructlab-pipelines-with-dsp-on-rhoai-.md

```diff
@@ -0,0 +1,646 @@
+model/config.json
+model/tokenizer.json
+model/tokenizer_config.json
+model/*.safetensors
+taxonomy/knowledge
+taxonomy/foundational_skills
+```
+
+[granite-7b-starter]: https://catalog.redhat.com/software/containers/rhelai1/granite-7b-starter/667ebf10abaa082bcf96ea6a
+[taxonomy tree]: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux_ai/1.2/html/creating_a_custom_llm_using_rhel_ai/customize_taxonomy_tree
+
+### Setting up Judge & Teacher model
+
+The Teacher model is used for Synthetic Data Generation (SDG) while the Judge model is used for model evaluation.
+
+For the Teacher model you need [mixtral-8x7b-instruct-v0-1] deployed with [skills-adapter-v3:1.2] and
+[knowledge-adapter-v3:1.2] LoRA layered skills and knowledge adapters.
+
+For the Judge model you will need the [prometheus-8x7b-v2-0 model].
+
+If you already have these models deployed you can skip the deployment steps and go straight to the secret set up for
+Judge and Teacher respectively.
+
+[mixtral-8x7b-instruct-v0-1]: https://catalog.redhat.com/software/containers/rhelai1/mixtral-8x7b-instruct-v0-1/6682619da4ea27a10a36e4c6
+[skills-adapter-v3:1.2]: https://catalog.redhat.com/software/containers/rhelai1/skills-adapter-v3/66a89d60ac8385376a3dabed
+[knowledge-adapter-v3:1.2]: https://catalog.redhat.com/software/containers/rhelai1/knowledge-adapter-v3/66a89ce222c498e45a8e0042
+[prometheus-8x7b-v2-0 model]: https://catalog.redhat.com/software/containers/rhelai1/prometheus-8x7b-v2-0/6682611224ba6617d472f451
+
+#### Deploy a judge model server (optional)
+
+Create a service account to be used for token authentication
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: judge-sa
+  namespace: <data-science-project-name/namespace>
+```
+
+Upload [prometheus-8x7b-v2-0 model] (Judge-Model) to the same object storage as before.
+
+For example using `ilab` to download and `s3cmd` to sync to object store you can do:
+```bash
+# You can also use Oras or Skopeo cli tools to download the model
+# If using other tools besides ilab, ensure that filenames are mapped
+# appropriately
+ilab model download --repository docker://registry.redhat.io/rhelai1/prometheus-8x7b-v2-0 --release 1.2
+
+# Default cache location for ilab model download is ~/.cache/instructlab/models
+s3cmd sync path/to/model s3://your-bucket-name/judge-model/
+```
+
+Navigate to the OpenShift AI dashboard
+* Choose Data Science Projects from the left hand menu and choose your data science project/namespace.
+* Choose the data connections tab, and click on the Add data connection button. Enter the details of your S3 bucket (object store) and click Add data connection.
+
+> [!NOTE]
+> Note: Before following the next step - Ensure that the `CapabilityServiceMeshAuthorization` status is `True` in `DSCinitialization` resource.
+
+Create a model server instance
+* Navigate to Data Science Projects and then the Models tab
+* On the right hand side select ‘Deploy model’ under Single-model serving platform
```

Comment:
```suggestion
* On the right hand side select **Deploy model** under Single-model serving platform
```

Reply:
Applicable to entire review: We should standardize on the "selectable"/ highlighted terms.  I'm using Bold (double asterisks), but we could use double quotes, italics, backticks, etc.